src/com/dabomstew/pkrandom/romhandlers/RomHandler.java

```diff
@@ -168,6 +170,9 @@ public void randomizeTrainerPokes(boolean usePowerLevels, boolean noLegendaries,
     public void typeThemeTrainerPokes(boolean usePowerLevels, boolean weightByFrequency, boolean noLegendaries,
             boolean noEarlyWonderGuard, int levelModifier);
 
+    public void legendaryTrainerPokes(boolean usePowerLevels, boolean weightByFrequency, boolean noLegendaries,
+        boolean noEarlyWonderGuard, int levelModifier);
```

Comment:
you should try to match style with existing code, which here means that you should indent like on line 171